common/persistence/persistenceMetricClients.go

```diff
@@ -436,6 +436,10 @@ func (p *workflowExecutionPersistenceClient) ListCurrentExecutions(
 	op := func() error {
 		var err error
 		resp, err = p.persistence.ListCurrentExecutions(ctx, request)
+		if err == nil && len(resp.Executions) == 0 {
+			p.metricClient.Scope(metrics.PersistenceEmptyResponseScope,
```

Comment:
I suggest reusing existing scopes, such as `metrics.PersistenceListCurrentExecutionsScope`. No need for new `metrics.OperationTag` as well since existing scope already indicates that.

Reply:
> I suggest reusing existing scopes, such as `metrics.PersistenceListCurrentExecutionsScope`. No need for new `metrics.OperationTag` as well since existing scope already indicates that.
Got it, I can reuse the scopes just need the new counter. Thanks for the advice!

Reply:
Got it, I can reuse the scopes just need the new counter. Thanks for the advice!